couchbase-entity-api/src/main/java/kaufland/com/coachbasebinderapi/schema/SchemaValidationLogger.kt

```diff
@@ -0,0 +1,12 @@
+package kaufland.com.coachbasebinderapi.schema
+
+interface SchemaValidationLogger {
```

Comment:
FYI: the SLF4J API also solves the problem of logging while giving the clients of the library the freedom to choose their logging implementation